src/ui/Combobox/ComboboxValues.ts

```diff
@@ -151,6 +151,37 @@ export class ComboboxValues {
     this.combobox.clearAll();
   }
 
+  private onScroll() {
+    const scrollEndReached = this.element.scrollTop + this.element.clientHeight >= this.element.scrollHeight;
+    if (scrollEndReached && this.combobox.options.scrollable.areMoreValuesAvailable()) {
+      this.combobox.onScrollEndReached();
+    }
+  }
+
+  public resetScroll() {
+    if (!this.isScrollable) {
+      return;
+    }
+    this.element.style.maxHeight = `${this.combobox.options.scrollable.maxDropdownHeight()}px`;
+    this.element.scrollTop = 0;
+  }
+
+  public moveActiveValueToTop() {
```

Comment:
The name of this function and the function below implies to me that the order of values is changed, would a name like `focusFirstValue` and `focusLastValue` be fitting?